PluginsAndFeatures/azure-toolkit-for-intellij/rider/ReSharper.Azure/src/Azure.Project/FunctionApp/FunctionAppProjectDetector.cs

```diff
@@ -34,8 +34,25 @@ namespace JetBrains.ReSharper.Azure.Project.FunctionApp
 {
     public static class FunctionAppProjectDetector
     {
-        private static readonly NugetId ExpectedPackageForNet50 = new NugetId("Microsoft.Azure.Functions.Worker");
-        private static readonly NugetId ExpectedPackageForOlder = new NugetId("Microsoft.NET.Sdk.Functions");
+        static class DefaultWorker
+        {
+            public static readonly NugetId ExpectedPackage = new NugetId("Microsoft.NET.Sdk.Functions");
```

Comment:
Could you please provide a more descriptive variable name, e.g. `FunctionNuGetPackageId`?